Multisig/UI/App/UIKit/UI Library/LoadableViewController/LoadableViewController.swift

```diff
@@ -37,12 +38,25 @@ class LoadableViewController: UIViewController {
             view.refreshControl = createRefreshControl()
         }
         setNeedsReload()
+
+        notificationCenter.addObserver(
```

Comment:
Although this behavior of reacting to safe changes is common to most of the loadable view controllers in the app, it is not all - Transaction details will not be dependent on the currently selected safe. 

Therefore, it is good to have this behavior in this parent class, however, I'd introduce a configuration variable that subclasses can override or set that would enable or disable reaction to the safe selection.

I would name this variable `reactsToSelectedSafeChanges`

Reply:
ok